### PR TITLE
Issue #17, renaming druidic to celtic_pagan

### PR DESCRIPTION
**Pull request by xirsoi**
Definitely need this looked over thoroughly. A couple of test games even, to be sure its all right.
Validator gave it an A-okay, but this is a big change across a LOT of files.

### CODE REVIEW

**Comment by rquinio** [comment]
Ok, I did a full review/diff and committed a few fixes.
Looks good to me ! 
I'll test this indirectly while working on the Circle of druids.

File: AncientReligions/localisation/ACR_druidic.csv [hunk 1]
```diff
@@ -11,10 +11,10 @@ GOD_LUGH;Lugh;;;;;;;;;;;;;x
...
+celtic_pagan;Celtic;;;;;;;;;;;;;x
+celtic_pagan_reformed;Druidic;;;;;;;;;;;;;x
+celtic_pagan_desc;Celtic paganism is one of the last surviving branch of the ancient druidic belief systems. A pantheon of gods and a world full of spirits, the children of Dannu rule the Skies, the Earth and the Otherworld. The main gods are Cernunnos, the Horned One, Dannu and Nuada, Angus, the god of Youth, Macha, the horse goddess, the great Daghda, Manannan mac Lir, the Sea God and Lugh of the Long Spear. Defining features in-game:\n�Y*�! Can raid neighbors for loot\n�Y*�! Low-tech non-pagans have very low Supply Limit in Pagan counties\n�Y*�! Can declare Subjugation Wars against all other pagans\n�Y*�! Rulers are limited to Gavelkind Succession\n�Y*�! Rulers cannot declare Holy Wars\n�Y*�! Inferior at conversion and vulnerable to non-pagan missionaries\n�Y*�! Rulers lose Prestige if at peace for too long\n�Y*�! Celtic Paganism is a feminist religion\n�Y*�! The Celtic Pagan faith can be Reformed;FRENCH;GERMAN;SPANISH;;;;;;;;;;x
```
Comment (rquinio) [inline AncientReligions/localisation/ACR_druidic.csv]: It needs to be _DESC for religions, otherwise the game doesn't find the localization

Comment (xirsoi) [inline AncientReligions/localisation/ACR_druidic.csv]: How odd, I'll get that fixed!

**Comment by rquinio** [comment]
I've fixed the _desc vs _DESC and resolved merge conflicts from develop.
Once merged I can create the PR for #16 ;)

**Pull request merged**